exercises/poker/src/test/java/PokerTest.java

```diff
@@ -6,158 +6,234 @@
-    }
-
-    @Ignore("Remove to run test")
-    @Test
-    public void flushVsFull() {
-        String flushTo8 = "3H 6H 7H 8H 5H";
-        String full = "4S 5H 4S 5D 4H";
-        assertEquals(Arrays.asList(full), new Poker(Arrays.asList(full, flushTo8)).getBestHands());
-    }
-
-    @Ignore("Remove to run test")
-    @Test
-    public void twoFulls() {
-        String fullOf4By9 = "4H 4S 4D 9S 9D";
-        String fullOf5By8 = "5H 5S 5D 8S 8D";
-        assertEquals(Arrays.asList(fullOf5By8), new Poker(Arrays.asList(fullOf4By9, fullOf5By8)).getBestHands());
-    }
-
-    @Ignore("Remove to run test")
-    @Test
-    public void fullVsSquare() {
-        String full = "4S 5H 4S 5D 4H";
-        String squareOf3 = "3S 3H 2S 3D 3H";
-        assertEquals(Arrays.asList(squareOf3), new Poker(Arrays.asList(full, squareOf3)).getBestHands());
-    }
-
-    @Ignore("Remove to run test")
-    @Test
-    public void twoSquares() {
-        String squareOf2 = "2S 2H 2S 8D 2H";
-        String squareOf5 = "4S 5H 5S 5D 5H";
-        assertEquals(Arrays.asList(squareOf5), new Poker(Arrays.asList(squareOf2, squareOf5)).getBestHands());
-    }
-
-    @Ignore("Remove to run test")
-    @Test
-    public void squareVsStraightFlush() {
-        String squareOf5 = "4S 5H 5S 5D 5H";
-        String straightFlushTo9 = "5S 7S 8S 9S 6S";
-        assertEquals(Arrays.asList(straightFlushTo9), new Poker(Arrays.asList(squareOf5, straightFlushTo9)).getBestHands());
-    }
-
-    @Ignore("Remove to run test")
-    @Test
-    public void twoStraightFlushes() {
-        String straightFlushTo8 = "4H 6H 7H 8H 5H";
-        String straightFlushTo9 = "5S 7S 8S 9S 6S";
-        assertEquals(Arrays.asList(straightFlushTo9), new Poker(Arrays.asList(straightFlushTo8, straightFlushTo9)).getBestHands());
-    }
-
-    @Ignore("Remove to run test")
-    @Test
-    public void threeHandWithTie() {
-        String spadeStraightTo9 = "9S 8S 7S 6S 5S";
-        String diamondStraightTo9 = "9D 8D 7D 6D 5D";
-        String threeOf4 = "4D 4S 4H QS KS";
-        assertEquals(Arrays.asList(spadeStraightTo9, diamondStraightTo9), new Poker(Arrays.asList(spadeStraightTo9, diamondStraightTo9, threeOf4)).getBestHands());
-    }
-
-    @Ignore("Remove to run test")
-    @Test
-    public void straightTo5AgainstAPairOfJacks() {
-        String straightTo5 = "2S 4D 5C 3S AS";
-        String twoJacks = "JD 8D 7D JC 5D";
-        assertEquals(Arrays.asList(straightTo5), new Poker(Arrays.asList(straightTo5, twoJacks)).getBestHands());
-    }
+	@Test
+	public void oneHand() {
+		String hand = "4S 5S 7H 8D JC";
+		assertEquals(Arrays.asList(hand), new Poker(Arrays.asList(hand)).getBestHands());
+	}
+
+	@Ignore("Remove to run test")
+	@Test
+	public void highestCardWins() {
+		String highest8 = "4D 5S 6S 8D 3C";
+		String highest10 = "2S 4C 7S 9H 10H";
+		String highestJ = "3S 4S 5D 6H JH";
+		assertEquals(Arrays.asList(highestJ), new Poker(Arrays.asList(highest8, highest10, highestJ)).getBestHands());
```

Comment:
This line is a bit long so I would add some linebreaks (120 chars is a common limit for how long a line should be). It's considered good practice to put each argument on a new line if an expression is too long to fit on one line. For example, in this case:
```
assertEquals(
    Arrays.asList(highestJ),
    new Poker(Arrays.asList(highest8, highest10, highestJ)).getBestHands()
);
```